docs/guides.md

```diff
@@ -483,3 +483,35 @@ Example with 4 GiB:
 Do not mess the units and make sure to set the same value as base value and as max value.
 
 Reboot to apply.
+
+### Autostart pool and virtual machines without HA
+
+We have two ways to sovle problem: CLI and Xen Orchestra.
+
+#### CLI:
+1. Determine the UUID of the pool for which we want to enable Auto Start. To do this, run the console command on the server:
+```
+# xe pool-list
+uuid ( RO) : 8a3b99c9-c4e9-802c-1298-794d939fb999
+```
+2. Allow autostart of virtual machines at the pool level with the found UUID command:
+`# xe pool-param-set uuid=8a3b99c9-c4e9-802c-1298-794d939fb999 other-config:auto_poweron=true`
+
+Now we enable autostart at the virtual machine level, for which: 
+3. Execute the command to get the UUID of the virtual machine:
+```
+# xe vm-list
+    uuid ( RO)           : 021fe476-dc06-23a7-b60e-092a3c286f70
+    name-label ( RW): VM
+    power-state ( RO): running
+```
+4. Enable autostart for each virtual machine with the UUID found: 
+`# xe vm-param-set uuid=021fe476-dc06-23a7-b60e-092a3c286f70 other-config:auto_poweron=true`
+
+5. Checking the output 
+`# xe vm-param-list uuid=021fe476-dc06-23a7-b60e-092a3c286f70 | grep other-config`
+
+#### Xen Orchestra
+
+In Xen Orchestra we can just enable a toggle in VM view. VM toggle in XO is automatically setting the pool.
+![](https://github.com/the29a/xcp-ng-org/blob/master/assets/img/screenshots/autopoweron1.png)
```

Comment:
You need to put the file inside the `asset` folder directly, and the link should be:

```
![](./assets/autopower1.png)
```

Reply:
Fixed. Kind of.

Reply:
Okay so you need to add the image in the PR. Now there's only the .md file modified.

Reply:
https://github.com/xcp-ng/xcp-ng-org/pull/76